### ISSUE TITLE
Secret Key Exposed

### ISSUE THREAD

**Issue by ironfroggy** [hydroshare/hydroshare#298]
https://github.com/hydroshare/hydroshare/blob/master/hydroshare/local_settings.py#L14

**Comment by mjstealey** [comment]
Resolved once release-candidate branch is merged into develop

**Comment by mjstealey** [comment]
new keys generated for all deployments
